trigger/twister3.py

```diff
@@ -91,8 +91,8 @@ def __init__(self):
         self.command_interval = 1
         self.incremental = None
         self.on_error = defer.Deferred()
-        self.todo = deque()
-        self.done = None
+        self.results_queue = defer.DeferredQueue(size=1024)
+        self._jobs = 0
```

Comment:
Is `self._jobs` just meant to be `False` here? I ask because in other contexts you're setting it to `True`. If so, can we please rename it to be something like `_jobs_pending` to improve readability?